frontend/src/lib/notification.tsx

```diff
@@ -8,14 +9,18 @@ export class Notification {
   url?: string;
   date: number | string = new Date().getTime();
   deleted: boolean = false;
+  cluster: string | null = getCluster();
 
-  constructor(message?: string, date?: number | string) {
+  constructor(message?: string, date?: number | string, cluster?: string | null) {
```

Comment:
Since all these are optional, maybe it's a good idea to make them an options object instead of parameters.

Reply:
updated